docs/security.md

```diff
@@ -37,26 +40,35 @@ Both stores must be base64 encoded, for example:
 
 **Note:** The base64 string of the keystore will probably be much longer than the snippet above, spanning 50 lines or so.
 
-With this and the password `secret` for the keystore and the private key, your JSON options file will look like this:
+Add the stores to your secrets in the DC/OS Secret store, for example if your base64 encoded keystores and truststores are server.jks.base64 and trust.jks.base64, respectively then do the following: 
```

Comment:
Add the stores to your secrets in the DC/OS secret store. For example, if your base64-encoded keystores and truststores are `server.jks.base64` and `trust.jks.base64`, respectively, then use the following commands to add them to the secret store: